tests/deprecated_api/test_remove_1-4.py

```diff
@@ -115,6 +115,33 @@ def test_v1_4_0_deprecated_metrics():
         iou(torch.randint(0, 2, (10, 3, 3)),
             torch.randint(0, 2, (10, 3, 3)))
 
+    from pytorch_lightning.metrics.functional.classification import recall
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        recall(torch.randint(0, 2, (10, 3, 3)),
+               torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional.classification import precision
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision(torch.randint(0, 2, (10, 3, 3)),
+                  torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional.classification import precision_recall
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision_recall(torch.randint(0, 2, (10, 3, 3)),
+                         torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional import precision
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision(torch.randint(0, 2, (10,)),
+                  torch.randint(0, 2, (10,)),
+                  class_reduction='micro')
+
+    from pytorch_lightning.metrics.functional import recall
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        recall(torch.randint(0, 2, (10,)),
+               torch.randint(0, 2, (10,)),
+               class_reduction='micro')
```

Comment:
Are these the same `precision` and `recall` being tested above?
Then I do not think it is nessesary to duplicate the test (even if the import pattern is different)

Reply:
I am testing the new precision and recall for the deprecated `class_reduction` parameter, if that answers your question?

Reply:
okay, sound like a good idea :]
could you provide a one-line comment about this?

Reply:
Done